code/source/playfab/PlayFabClientApi_c.cpp

```diff
@@ -0,0 +1,164 @@
+#include <stdafx.h>
+#include <playfab/AsyncProvider.h>
+#include <playfab/PlayFabClientApi.h>
+#include <playfab/PlayFabBaseModel.h>
+
+using namespace PlayFab;
+using namespace PlayFab::ClientModels;
+
+HRESULT PlayFabInitialize(
+    _Outptr_ PlayFabStateHandle* stateHandle
+) noexcept

```

Comment:
>noexcept [](start = 2, length = 8)

is the expectation that the internal code will make sure no exceptions are throw across the API boundary? or should this API boundary file be in charge of enforcing that?

I guess I don't actually expect that this prototype has all the code in place to hide the potential exceptions and these `noexcept` declarations are enough to at least suggest that's still part of the plan. but I thought I'd ask anyway

Reply:
Ya there may be exceptions that make it to the boundary currently, but its eventually the plan to catch and translate them all. 

---
In reply to: [596375499](https://github.com/jplafonta/XPlatCppSdk/pull/2#discussion_r596375499) [](ancestors = 596375499)